src/jit/morph.cpp

```diff
@@ -7001,24 +7018,85 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
             {
 #if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
 
+                // hasMultiByteArgs will determine if the struct can be passed
+                // in registers. If it cannot we will break the loop and not
+                // fastTailCall.
```

Comment:
Even with this added comment, this name is misleading - it is assigning it the negation of the value returned by a method that sounds like it ought to be at least a subset of this condition - i.e. if `VarTypeIsMultiByteAndCanEnreg` returns true, I would expect `hasMultiByteArgs` to be true. It seems that `hasMultiByteArgs` should really be something like `hasMultiByteStackArgs`.